util/chunk/chunk.go

```diff
@@ -609,6 +609,16 @@ func (c *Chunk) Column(colIdx int) *Column {
 	return c.columns[colIdx]
 }
 
+// SetCol sets the colIdx Column to col and returns the old Column.
+func (c *Chunk) SetCol(colIdx int, col *Column) *Column {
+	if col == c.columns[colIdx] {
```

Comment:
Will this line be hit?

Reply:
It's for safety to avoid some alive Columns are recycled by users.